packages/manager/apps/dedicated/client/app/cdn/dedicated/manage/domains/index.js

```diff
@@ -4,18 +4,39 @@ import 'oclazyload';
 
 const moduleName = 'ovhManagerCdnDomainsLazyLoading';
 
-angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
-  /* @ngInject */ ($stateProvider) => {
-    $stateProvider.state('app.networks.cdn.dedicated.manage.domain.**', {
-      url: '/domain',
-      lazyLoad: ($transition$) => {
-        const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+angular
+  .module(moduleName, ['ui.router', 'oc.lazyLoad'])
+  .config(
+    /* @ngInject */ ($stateProvider) => {
+      $stateProvider.state('app.networks.cdn.dedicated.manage.domain', {
+        url: '/domain',
+        redirectTo: 'app.networks.cdn.dedicated.manage.domain.index',
+        views: {
+          cdnView: {
+            template: '<div ui-view></div>',
+          },
+        },
+        resolve: {
+          breadcrumb: /* @ngInject */ ($translate) =>
+            $translate.instant('cdn_dedicated_domains'),
+        },
+      });
+
+      $stateProvider.state(
+        'app.networks.cdn.dedicated.manage.domain.index.**',
+        {
+          url: '',
+          lazyLoad: ($transition$) => {
+            const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
+
+            return import('./domains.module').then((mod) =>
+              $ocLazyLoad.inject(mod.default || mod),
```

Comment:
👍

Reply:
@marie-j I had done repsac migration for CDA. QA was also doe but its not merged to develop. I will release my branch on this and submit for QA again.